src/contentsharecontroller/DefaultContentShareController.ts

```diff
@@ -83,7 +96,7 @@ export default class DefaultContentShareController
   }
 
   stopContentShare(): void {
-    this.audioVideo.stop();
+    this.contentAudioVideo.stop();
```

Comment:
A general question, does videoTileWasRemoved get fired when stopContentShare is called ?  should we call another bindStream here?